python/doc/theory/data_analysis/graphical_fitting_test.rst

```diff
@@ -105,47 +82,29 @@ irrelevant.
     View(graph)
 
 
-**Henry’s line**
+Henry’s line
```

Comment:
In non-French documents, this is known as the "Normal probability plot":

https://en.wikipedia.org/wiki/Normal_probability_plot

Reply:
yes... I added the non french name!